PWGJE/Tasks/jetSpectraEseTask.cxx

```diff
@@ -166,17 +206,53 @@ struct JetSpectraEseTask {
 
       float dPhi = RecoDecay::constrainAngle(jet.phi() - vPsi2, -o2::constants::math::PI);
       registry.fill(HIST("hdPhi"), dPhi);
-      registry.fill(HIST("hJetPtdPhiq2"), jetpT_bkgsub, dPhi, qPerc[0]); /* check the dphi */
+      registry.fill(HIST("hCentJetPtdPhiq2"), originalCollision.centFT0C(), jetpT_bkgsub, dPhi, qPerc[0]);
     }
     registry.fill(HIST("hEventCounter"), counter++);
+
+    if (originalCollision.centFT0C() < CentRange->at(0) || originalCollision.centFT0C() > CentRange->at(1)) /* for counting */
+      return;
+    registry.fill(HIST("hEventCounter"), counter++);
   }
   PROCESS_SWITCH(JetSpectraEseTask, processESEDataCharged, "process ese collisions", true);
 
+  void processESEOccupancy(soa::Join<aod::JetCollisions, aod::JCollisionPIs>::iterator const& collision,
+                           soa::Join<aod::JetTracks, aod::JTrackPIs> const& tracks,
+                           soa::Join<aod::Tracks, aod::TracksExtra, aod::TracksDCA, aod::TrackSelection> const&,
+                           soa::Join<aod::Collisions, aod::CentFT0Cs, aod::Qvectors, aod::QPercentileFT0Cs> const&)
+  {
+    float count{0.5};
+    registry.fill(HIST("hEventCounterOcc"), count++);
+    const auto orgCol = collision.collision_as<soa::Join<aod::Collisions, aod::CentFT0Cs, aod::Qvectors, aod::QPercentileFT0Cs>>();
```

Comment:
here you dont need the centFT0C table. I think you can probably even join aod::Qvectors, aod::QPercentileFT0Cs to aod::JetCollisions and then not have this original collision part at all. Then do everything on that collision table

Reply:
Hi @nzardosh,
Thanks! I have removed/changed the originalCollision, and it does indeed still work. I suppose in that case I also don't need aod::JCollisionPIs? (It' still there for now)
Cheers,
Joachim

Reply:
Thanks alot for removing it. Yes you can also remove aod::JCollisionPIs.

Thanks,
Nima